content/docs/started/k8s/kfctl-existing-arrikto.md

```diff
@@ -259,16 +130,19 @@ deploy Kubeflow:
 
 ### Log in as a static user
 
-After deploying Kubeflow, the Kubeflow dashboard is available at the Istio Gateway IP.
-To get the Istio Gateway IP, run:
+The default way of accessing Kubeflow is via port-forward.
+This enables you to get started quickly without imposing any requirements on your environment.
 
 ```bash
-kubectl get svc -n istio-system istio-ingressgateway -o jsonpath='{.status.loadBalancer.ingress[0].ip}'
+# Kubeflow will be available at localhost:8080
+kubectl port-forward svc/istio-ingressgateway -n istio-system 8080:80
 ```
 
-Get the IP and open it in a browser: `https://<LoadBalancerIP address>/`.
+The credentials of the default user are `admin@kubeflow.org`:`12341234`.
```

Comment:
Same comment as above, about credentials displayed on the website.

Reply:
Will add warning and prompt user to change them.